### PR TITLE
Add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by bruk19**
In this PR: 

- Created class person
- Constructed with `name`, `age`, and `parent_permission` as parameter. 
- Added `Getters` for `@id`, `@name`, and `@age`.
- Added `Setters` for `@name` and `@age`.
- Added Private method `is_of_age?`
- Added Public method `can_use_services`
- Created class `Student` inherit from `person`
- Constructor extends parent's constructor by adding `@classroom`
- Added Method `play_hooky`
- Created class `Teacher` inherit from `person`
- Constructor extends parent's constructor by adding `@specialization`
- Added Override `can_use_services?`

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# create a person class 
+
+class person
```
Comment (prolajumokeoni) [inline person.rb]: - [ ] Kindly capitalize the `p` in `Person` as this is a class name 👍🏿

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# create a student class 
+class student < person
```
Comment (prolajumokeoni) [inline student.rb]: - [ ] Kindly capitalize the `s` in Student and `p` in Person as they are both class names 👍🏿

File: Teacher.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# create a Teacher class
+class Teacher < person
```
Comment (prolajumokeoni) [inline Teacher.rb]: - [ ] Kindly capitalize the `p` in `Person` as this is a class name 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @bruk19 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @bruk19 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**